ffi/capi/Cargo.toml

```diff
@@ -56,6 +56,9 @@ x86tiny = ["dlmalloc"]
 # Enables no_std builds for freertos
 freertos = ["freertos-rust", "cortex-m"]
 
+# Enables things that enable Rust to be linked to C++ (etc) in WASM
+wasm_embedding = []
```

Comment:
Nit: The only reason this is building right now is because it is gated on wasm32.  However, your global allocator isn't strictly about wasm32; it just calls down to four C function imports.  Therefore, consider promoting your new global allocator to the top level and do the mutually-exclusive dance with the other two global allocator features.  Alternatively, move this feature out of the "Only 1 of the following features can be set" section.

Reply:
I expect there to be more things to be hooked up here, but you're right that the alloc stuff should be its own feature.